### DISCUSSION TITLE
Pin Input Component

### DISCUSSION THREAD

**Discussion by whoisandy** [mantinedev/mantine#3128] [Feature requests]
First off thanks for an amazing component library! It would be cool to add a `PinInput` component to the core package as well. Could be used to capture any one time passwords or verification codes used to login during auth or 2fa process? 

Check out a working [example here](https://hqwr3g.csb.app). The code to the `PinInput` component can be found [here](https://codesandbox.io/s/focused-monad-hqwr3g?file=/src/PinInput.tsx).

This example above has a bare minimum code to explore the functionality needed for a pin input, similar the [pin input component](https://chakra-ui.com/docs/components/pin-input) in `@chakra-ui` framework. 

![CleanShot 2022-12-06 at 20 10 03](https://user-images.githubusercontent.com/297132/205941761-f0653713-075a-48e5-9724-d6c55a8178eb.gif)

**Comment by rtivital** [comment]
https://mantine.dev/labs/pin-input/

**Reply by whoisandy** [reply]
Awesome! How do I browse all the components in the labs section? Didn't find a link on the mantine docs site?

**Reply by rtivital** [reply]
You cannot do that. You can find components via source code.
